File: modul 2/python/PRAK205_2410817110008_Muhammad Azma Al Faqih.py
sisiA = int(input())
sisiB = int(input())
sisiC = ((sisiB * sisiB)-(sisiA*sisiA))**0.5
keliling = sisiA + sisiB + sisiC
luas = 0.5 * sisiC * sisiA
print(f"Alas = {sisiC:.0f} cm")
print(f"Tinggi = {sisiA:.0f} cm")
print(f"Keliling = {keliling:.0f} cm")
print(f"Luas = {luas:.0f} cm^2")

File: modul 2/c/PRAK205-2410817110008-Muhammad Azma Al Faqih.c
#include <math.h>
#include <stdio.h>

int main() {
  int sisiA, sisiB, keliling, sisiC, luas;
  scanf("%d %d", &sisiA, &sisiB);

  sisiC = sqrt(sisiB * sisiB - sisiA * sisiA);
  keliling = sisiA + sisiB + sisiC;
  luas = 0.5 * sisiC * sisiA;

  printf("Alas = %d cm\n", sisiC);
  printf("Tinggi = %d cm\n", sisiA);
  printf("Keliling = %d cm\n", keliling);
  printf("Luas = %d cm^2\n", luas);
}
